lua/entities/gmod_wire_expression2/core/custom/sv_rune2.lua

```diff
@@ -20,69 +21,54 @@ local function buildBody(args) -- Why does the wireteam do this
 end
 
 -- We use this for try and catch
-local function runE2InstanceSafe(compiler,func,body) -- Varargs to pass to the e2 function
-    -- Will always return pcallerror,errstr first even if it didn't error.
-    local args = {pcall(func,compiler,body)}
+local function runE2InstanceSafe(compiler,func,body,...)
```

Comment:
I will try to add vararg support for the `try`, so this is just the initial part of it I guess.